content/pt-br/docs/reference/setup-tools/kubeadm/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Kubeadm"
+weight: 10
+no_list: true
+content_type: conceito
+card:
+  name: reference
+  weight: 40
+---
+
+<img src="/images/kubeadm-stacked-color.png" align="right" width="150px">O Kubeadm é uma ferramenta criada para fornecer o `kubeadm init` e o `kubeadm join` como "caminhos rápidos" de melhores práticas para criar clusters Kubernetes.
+
+O kubeadm executa as ações necessárias para colocar um cluster minimamente viável em funcionamento. Por propósito, ele se preocupa apenas com bootstrapping, não com provisionamento de máquinas. Da mesma forma, a instalação de vários complementos, como o Kubernetes Dashboard, soluções de monitoramento e complementos específicos da nuvem, não está no escopo.
```

Comment:
não está no escopo.
could be plural "estão"

Reply:
I think it's singular, because the direct sentence is "a instalação não está no escopo". Are you sure that?

Reply:
Agree that "não está no escopo" should be singular since it refers to "instalação".